recipes/libpqxx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,14 @@
+#include <cstdlib>
+#include <iostream>
+
+#include "pqxx/pqxx"
+
+int main()
+{
+    std::cout << "libpqxx version: " << PQXX_VERSION << std::endl;
+
+    pqxx::connection conn("postgres://tpilupsd:7hJFAnmfyuD27orImLVfKQX1mdMUJTq_@stampy.db.elephantsql.com:5432/tpilupsd");
```

Comment:
Just asking:
Does this test actually connect to the internet/network?
Can this test time out because of connection issues?

Reply:
Yes. This code connect to ElephantSQL: https://www.elephantsql.com/


The code was added a few years ago, but was kept because it was a change of Bincrafters members: https://github.com/bincrafters/conan-libpqxx/commit/b984924a15a3a5f32dc144b973107707215b2007

Should remove it?

Reply:
My issue with this is that connection to external networks can be non-secure and the time to connect can be either very fast or very slow.
Unless this is just a constructor and no connection is actually made.

Reply:
The constructor makes an external connection.

I remove this code.

Reply:
Removed: f19309d743b6c3be46e5a50cbe1dbd3a4e1f50f4